core/src/main/java/org/keycloak/representations/workflows/WorkflowRepresentation.java

```diff
@@ -146,6 +147,20 @@ public void setState(WorkflowStateRepresentation state) {
         this.state = state;
     }
 
+    @Override
+    public boolean equals(Object obj) {
+        if (obj == this) {
+            return true;
+        }
+        if (!(obj instanceof WorkflowRepresentation)) {
+            return false;
+        }
+        WorkflowRepresentation that = (WorkflowRepresentation) obj;
+        // TODO: include state in comparison?
```

Comment:
Status is a server-side information and not updatable by users. At least so far ...